errata_tool/release.py

```diff
@@ -164,3 +174,11 @@ def create(klass, name, product, product_versions, type, program_manager,
             print(body)
             raise ReleaseCreationError('see field_errors <div>')
         return klass(name=name)
+
+
+def unquote_plus(string):
+    """
+    This method is similar to urllib.parse.unquote_plus(), but it *only*
```

Comment:
whitespace error in doctext.  doctext entries should not include leading whitespace

"""This method

Reply:
I understand why the standard says this, but to me this is less readable! I'll update it.